packages/p2p/src/components/advertiser-page/advertiser-page.scss

```diff
@@ -91,38 +94,120 @@
                 border-right: 1px solid #ededed;
             }
         }
+
+        @include mobile {
+            width: 110vw;
+        }
+
+        &--wrapper {
+            display: flex;
+            align-items: center;
+        }
+
+        .dc-themed-scrollbars__autohide--is-hovered::-webkit-scrollbar-thumb {
+            display: none;
+        }
+    }
+    &__cell {
+        @include mobile {
+            display: block;
+        }
+        &-limit {
+            padding-top: 0.8rem;
+        }
     }
     &__popover {
         &-icon {
-            margin: 1.6rem 0.8rem;
+            margin: 1.6rem 0.8rem 1.6rem 2.4rem;
+
+            @include mobile {
+                margin: 1.6rem 1rem;
+            }
         }
     }
     &__adverts {
         &-button {
             align-items: center;
             display: flex;
             justify-content: center;
+
+            @include mobile {
+                justify-content: flex-end;
+
+                .dc-btn__small {
+                    height: 4rem;
+                }
```

Comment:
Instead of overriding the `<Button small>` button, why not use a `large` prop for the `<Button>` component for mobile devices? i.e. `<Button large>`

Reply:
Done.